benchmarks/bench_plot_incremental_pca.py

```diff
@@ -116,9 +111,8 @@ def variable_batch_size_comparison(data):
         all_times = defaultdict(list)
         all_errors = defaultdict(list)
         pca = PCA(n_components=n_components)
-        rpca = RandomizedPCA(n_components=n_components, random_state=1999)
-        results_dict = {k: benchmark(est, data) for k, est in [('pca', pca),
-                                                               ('rpca', rpca)]}
+        rpca = PCA(n_components=n_components, svd_solver='randomized', random_state=1999)
+        results_dict = {k: benchmark(est, data) for k, est in [('pca', pca)]}
```

Comment:
This example does not work.
We should either keep `rcpa` here or remove it everywhere.

Reply:
@TomDLT Is the suggestion of @jnothman a few lines above to use `PCA(svd_solver='randomized')` what you mean with keeping `rpca` ?

Reply:
Yes, or we can just drop it.
My comment was meant to mention that the example is currently broken.